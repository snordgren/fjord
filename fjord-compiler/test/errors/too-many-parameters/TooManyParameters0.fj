module test.errors.TooManyParameters0

myVal : Int 
myVal x =
  0
