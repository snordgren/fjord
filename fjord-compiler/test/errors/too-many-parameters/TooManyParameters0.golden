errors/too-many-parameters/TooManyParameters0.fj:4:7:
  |
4 | myVal x =
  |       ^^^^
too many parameters, expected 0
